how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -0,0 +1,175 @@
+# How to use Kata Containers and "cri" (contaienrd plugin)
+## Requiriments 
+- Kuberentes, kubelet 1.10+
+- cri-containerd 1.1.0+
+- Kata Containers 0.2.0+
+
+## Run all this
+```
+#sed -n '/^```/,/^```/ p'  | sed '/^```/ d' > script.sh
+```
+
+## Install cri
+
+Follow the instructions from "cri" [installation guide](http://github.com/containerd/cri/blob/master/docs/installation.md)
+
+### Quick installation.
+```
+# Install cri-containerd
+VERSION="1.1.0"
+curl -OL \
+https://storage.googleapis.com/cri-containerd-release/cri-containerd-${VERSION}.linux-amd64.tar.gz
+sudo tar -C / -xzf cri-containerd-${VERSION}.linux-amd64.tar.gz
+
+sudo apt-get update
+sudo apt-get install libseccomp2
+sudo systemctl daemon-reload
+sudo systemctl start containerd
+
+```
+
+## Install Kata Containers / Kata
+
+Follow the instructions to install Kata in your system:
+
+  * [Fedora*](install/fedora-installation-guide.md)
+  * [Ubuntu*](install/ubuntu-installation-guide.md)
+
+Example (Ubuntu*):
```

Comment:
Sorry, if this is not useful but I would like to make this doc ready to test - so I am adding this *examples* in ubuntu to have a happy default path to tests :P

Reply:
Yes, I understand. But if we document how to install Kata for Ubuntu in two different locations, one of them is bound to get "stale" eventually.

This might be getting a bit baroque, but if you put together the mini doc explaining how to "execute" a document, this step *could* become:

```bash
$ echo "Install kata using https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md"
$ echo "press return when done" && read junk
```

But my view on this may change after I've had a cup of coffee :smile: